tests/scansion_test.py

```diff
@@ -170,7 +170,7 @@ def test_aen_1_254(self):
         text = "Ollī subrīdēns hominum sator atque deōrum"
         verse = self.scan_verse(text)
         self.assertEqual(
-            verse.var_pron, "olliː subriːdeːns hominũː sato ratkwe deoːrũː"
+            verse.var_pron, "olliː subriːdẽːs hominũː sato ratkwe deoːrũː"
```

Comment:
Changed this make it consistent with the n nasalization fix I did earlier in this PR